src/controllers/signup.js

```diff
@@ -0,0 +1,38 @@
+const { addUser } = require('../database/queries/addData');
+
+exports.get = (req, res) => {
+  res.render('signup', {
+    layout: 'login_signup',
+    title: 'signup ',
+    js: ['collectData', 'signup'],
+    css: ['signup'],
+  });
+};
+
+exports.post = (req, res) => {
+  //   req.body = {
+  //   firstSection: {
+  //     firstname: 'Angham',
+  //     lastname: 'Aabed',
+  //     mobile_number: '0097234567',
+  //     email: 'a@gmail.com',
+  //   },
+  //   secondSection: {
+  //     specalization_id: 1,
+  //     freelancer_url: 'https://github.com/angham',
+  //     photo_url: 'https://www.iconspng.com/image/36709/face-avatar-man-male-handsome-3',
+  //   },
+  //   thirdSection: {
+  //     password: 'a123456',
+  //   }
+  // }
+  
+  // const userData = req.body;
+  // addUser(userData)
+  // .then(result =>{
+  // res.send(JSON.stringify({ success : 'Regestration success ...'}));
+  // })
+  // .catch( err =>{
+  //   res.send(JSON.stringify({ error : 'There is error in networke '}));
+  // })
+}
```

Comment:
why you pull the comments? 
and add a new line at the last of the file.

Reply:
@Angham116  look to label  ? 
I think it is in progress